collections/Quests/MultipleChoice/EditView.tsx

```diff
@@ -9,18 +8,30 @@ const EditView = () => {
   const question = useQuestStore(state => state.question)
   const setQuestion = useQuestStore(state => state.setQuestion)
 
-  const correctAnswer = useQuestStore(state => state.correctAnswer)
-  const setCorrectAnswer = useQuestStore(state => state.setCorrectAnswer)
+  const rightAnswers = useQuestStore(state => state.rightAnswers)
+  const setRightAnswers = useQuestStore(state => state.setRightAnswers)
+
+  const correctAnswers = useQuestStore(state => state.correctAnswers)
+  const setCorrectAnswers = useQuestStore(state => state.setCorrectAnswers)
 
   const wrongAnswers = useQuestStore(state => state.wrongAnswers)
   const setWrongAnswers = useQuestStore(state => state.setWrongAnswers)
 
-  const shuffledAnswers = useQuestStore(state => state.shuffledAnswers)
   const setShuffledAnswers = useQuestStore(state => state.setShuffledAnswers)
 
   const intl = useIntl()
 
-  const replaceAnswer = (
+  const label1 = intl.formatMessage({
+    id: 'quests.multipleChoice.editView.labelTask',
+  })
+  const label2 = intl.formatMessage({
+    id: 'quests.multipleChoice.editView.labelCorrectAnswer',
+  })
+  const label3 = intl.formatMessage({
+    id: 'quests.multipleChoice.editView.labelWrongAnswer',
+  })
```

Comment:
make these inline since they are only used once

Reply:
Fixed in 066c5c5